src/OrchardCore.Modules/OrchardCore.ContentTypes/Controllers/AdminController.cs

```diff
@@ -170,6 +170,11 @@ public async Task<ActionResult> EditPOST(string id, EditTypeViewModel viewModel)
             viewModel.DisplayName = contentTypeDefinition.DisplayName;
             viewModel.Editor = await _contentDefinitionDisplayManager.UpdateTypeEditorAsync(contentTypeDefinition, this);
 
+            if(IsAlphaNumeric(viewModel.Settings["ContentTypeSettings"].Value<string>("Stereotype")))
```

Comment:
- Format
- The check is the opposite
- The error message is wrong

Reply:
Nah, the keyboard made me crazy :) , just a sec